sdk/include/psdk/shellapi.h

```diff
@@ -281,12 +279,19 @@ typedef struct _NOTIFYICONDATAW {
 #endif
 } NOTIFYICONDATAW,*PNOTIFYICONDATAW;
 
+/* NOTIFYICONDATA sizes to force older than current structs */
```

Comment:
Such comment is unnecessary. Such constructs are well known by any win32 developer. See for example,
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=V%28%5B0-9%5D%2B%29_SIZE&sr=1

Reply:
Removed.